lisa/operating_system.py

```diff
@@ -372,7 +372,9 @@ def _package_exists(self, package: str, signed: bool = True) -> bool:
 
     def _get_os_version(self) -> OsVersion:
         os_version = OsVersion("")
-        cmd_result = self._node.execute(cmd="lsb_release -as", no_error_log=True)
+        cmd_result = self._node.execute(
+            cmd="lsb_release -ar 2>/dev/null", shell=True, no_error_log=True
```

Comment:
Add reason here, why it needs to redirect errors to null. If there is errors, how we know the reason?